### DISCUSSION TITLE
Blue LED always on when battery is in but switch is off. Anyway to turn it off to save power?

### DISCUSSION THREAD

**Discussion by toingee** [Xinyuan-LilyGO/T-Energy-S3#5] [Q&A]
Blue LED always on when battery is in but switch is off. I was wondering Anyway to turn it off to save power?

**Comment by SHKaminski** [comment]
I think I have the same question. With other boards with a built-in battery holder, the on/off switch will cut the power to the entire device, but (I think) maintain charging if the USB in connected. 
However, with the T-Energy-S3, the switch apparently does nothing, unless I'm missing something.
